index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
```

Comment:
Reminder to note this as a breaking change. With this the `opts` on `osm.get()` does nothing and is not passed through to hyperkv as the docs currently state.